### PR TITLE
Fix tinyusb lost written chars

### PR DESCRIPTION
**Pull request by GMagician**
This will fix lost chars when tx buffer is full

Address #163

### CODE REVIEW

**Comment by ladyada** [comment]
@hathach please take a look
@GMagician do you have a test sketch for htis issue?

**Comment by GMagician** [comment]
yes! and it fix my lost sent char

**Comment by GMagician** [comment]
rebased to resolve conflit

File: cores/arduino/Adafruit_TinyUSB_Core/tinyusb/src/class/cdc/cdc_device.h [hunk 1]
```diff
@@ -168,11 +167,6 @@ static inline bool tud_cdc_peek (int pos, uint8_t* u8)
   return tud_cdc_n_peek(0, pos, u8);
 }
 
-static inline uint32_t tud_cdc_write_char (char ch)
```
Comment (hathach) [inline cores/arduino/Adafruit_TinyUSB_Core/tinyusb/src/class/cdc/cdc_device.h]: please keep this function, it is part of the API of the tinyusb stack, it is OK to be unused

Comment (GMagician) [reply cores/arduino/Adafruit_TinyUSB_Core/tinyusb/src/class/cdc/cdc_device.h]: remove commit that change .h so this should sadisfy your request

**Pull request merged**